# Message personnalisÃ© pour le statut des ennemis
string name
float32 x
float32 y
string state  # alive, eliminated, spawning
